client/src/Movies/SavedList.js

```diff
@@ -12,7 +13,7 @@ export default class SavedList extends Component {
         {this.props.list.map(movie => (
           <span className="saved-movie">{movie.title}</span>
         ))}
-        <div className="home-button">Home</div>
+        <Link to="/"><div className="home-button">Home</div></Link>
```

Comment:
The inner divs only purpose is to apply a className, you can put the className on Link